frontend/src/components/LoginPage.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from "react";
+import TUMLogo from "../assets/tum-logo.svg"; // 确保路径正确
+
+const LoginPage = () => {
+    const [email, setEmail] = useState("");
+    const [error, setError] = useState("");
+
+    const handleSignIn = () => {
+        const emailRegex = /^[^\\s@]+@[^\\s@]+\\.[^\\s@]+$/;
+        if (!emailRegex.test(email)) {
+            setError("Invalid Account, please try again!");
+        } else {
+            setError("");
+            console.log("Sign-in with email:", email);
+        }
+    };
+
+    return (
+        <div
+            className="relative flex justify-center items-center min-h-screen"
+            style={{
+                backgroundColor: "#f5f5f5", // 设置页面背景色
```

Comment:
Maybe you can use bg-[#f5f5f5], the tailwindCSS in className instead of inline style

Reply:
Other parts are the same

Reply:
ok